spark/src/test/scala/org/apache/comet/CometExpressionSuite.scala

```diff
@@ -2002,10 +2002,21 @@ class CometExpressionSuite extends CometTestBase with AdaptiveSparkPlanHelper {
     Seq(true, false).foreach { dictionaryEnabled =>
       withTempDir { dir =>
         val path = new Path(dir.toURI.toString, "test.parquet")
-        makeParquetFileAllTypes(path, dictionaryEnabled = dictionaryEnabled, 10000)
+        makeParquetFileAllTypes(path, dictionaryEnabled = dictionaryEnabled, 1000)
```

Comment:
What is the reason for the change from `10000` to `1000`?

Reply:
I had to scroll up really far while debugging the empty string vs null issue which dumped all 10k records to the terminal 😂 happy to change it back, not sure how much it matters to do 10k rows or not.